samples/Calling/src/app/views/HomeScreen.tsx

```diff
@@ -284,10 +302,21 @@ export const HomeScreen = (props: HomeScreenProps): JSX.Element => {
                     label={'Teams user ID'}
                     required
                     placeholder={'Enter a Teams user ID'}
```

Comment:
Maybe we can also change the placeholder to 'Enter a Teams id (8:orgid:<UUID>)'

Reply:
updated